### DISCUSSION TITLE
Hello, Google Play can't be downloaded in the same way. It's sleeping in the wallet device, nothing, nothing happened. It doesn't work properly on the Internet. vpn It doesn't work at all, I don't know what to do, come to the factory, extraction is going on

### DISCUSSION THREAD

**Discussion by kambyzkargrshhamt** [community/community#34698] [Other Feature Feedback, Questions, & Ideas]
The Internet is so slow that Google doesn't come up. The drip of the Internet, by the grace of God, the root of this mullahs will be removed.I stayed on my phone. Google Play engine can't download anything. How cruel is it to us, our children, in Canada. Look at the houses they buy for more than 20 million. God, destroy this generation of houses.

**Comment by kambyzkargrshhamt** [comment]
Google is giving error 404. Now, error 404 has appeared on your application

**Comment by queenofcorgis** [comment]
Hi @kambyzkargrshhamt,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Google. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
